src/modules/worlds/actions.ts

```diff
@@ -15,3 +16,114 @@ export const fetchWorldsWalletStatsFailure = (address: string, error: string) =>
+  action(GET_WORLD_PERMISSIONS_SUCCESS, { worldName, permissions })
+export const getWorldPermissionsFailure = (worldName: string, error: string) => action(GET_WORLD_PERMISSIONS_FAILURE, { worldName, error })
+
+export type GetWorldPermissionsRequestAction = ReturnType<typeof getWorldPermissionsRequest>
+export type GetWorldPermissionsSuccessAction = ReturnType<typeof getWorldPermissionsSuccess>
+export type GetWorldPermissionsFailureAction = ReturnType<typeof getWorldPermissionsFailure>
+
+// Post World Permissions Type
+export const POST_WORLD_PERMISSIONS_REQUEST = '[Request] Post World Permissions'
+export const POST_WORLD_PERMISSIONS_SUCCESS = '[Success] Post World Permissions'
+export const POST_WORLD_PERMISSIONS_FAILURE = '[Failure] Post World Permissions'
+
+export const postWorldPermissionsRequest = (
+  worldName: string,
+  worldPermissionNames: WorldPermissionNames,
+  worldPermissionType: WorldPermissionType
+) => action(POST_WORLD_PERMISSIONS_REQUEST, { worldName, worldPermissionNames, worldPermissionType })
+export const postWorldPermissionsSuccess = (
+  worldName: string,
+  worldPermissionNames: WorldPermissionNames,
+  worldPermissionType: WorldPermissionType
+) => action(POST_WORLD_PERMISSIONS_SUCCESS, { worldName, worldPermissionNames, worldPermissionType })
+export const postWorldPermissionsFailure = (
+  worldName: string,
+  worldPermissionNames: WorldPermissionNames,
+  worldPermissionType: WorldPermissionType,
+  error: string
+) => action(POST_WORLD_PERMISSIONS_FAILURE, { worldName, worldPermissionNames, worldPermissionType, error })
+
+export type PostWorldPermissionsRequestAction = ReturnType<typeof postWorldPermissionsRequest>
+export type PostWorldPermissionsSuccessAction = ReturnType<typeof postWorldPermissionsSuccess>
+export type PostWorldPermissionsFailureAction = ReturnType<typeof postWorldPermissionsFailure>
+
+// Put World Permissions Type
+export const PUT_WORLD_PERMISSIONS_REQUEST = '[Request] Put World Permissions'
+export const PUT_WORLD_PERMISSIONS_SUCCESS = '[Success] Put World Permissions'
+export const PUT_WORLD_PERMISSIONS_FAILURE = '[Failure] Put World Permissions'
+
+export const putWorldPermissionsRequest = (
+  worldName: string,
+  worldPermissionNames: WorldPermissionNames,
+  worldPermissionType: WorldPermissionType.AllowList,
+  newData: string
+) => action(PUT_WORLD_PERMISSIONS_REQUEST, { worldName, worldPermissionNames, worldPermissionType, newData })
+export const putWorldPermissionsSuccess = (
+  worldName: string,
+  worldPermissionNames: WorldPermissionNames,
+  worldPermissionType: WorldPermissionType.AllowList,
+  newData: string
+) => action(PUT_WORLD_PERMISSIONS_SUCCESS, { worldName, worldPermissionNames, worldPermissionType, newData })
+export const putWorldPermissionsFailure = (
+  worldName: string,
+  worldPermissionNames: WorldPermissionNames,
+  worldPermissionType: WorldPermissionType.AllowList,
+  newData: string,
```

Comment:
It seems that these parameters are not used in the reducer, would you mind removing them?
```suggestion
```
It seems that there are other actions with the same issue, would you mind checking them as well?